nasa/earthdata-search
# EDSC-3563: Display Granule Cards without Temporal Information

# Overview

### What is the feature?

Given a dataset has granules with neither a start nor end date, when a user displays collection details the granule highlight cards will show "Not Provided" for the dates vs. the page breaking which it currently does.  For consistency, the cards in the granule list view was updates to show "Not Provided" versus an empty string.

### What is the Solution?

If temporal information is undefined, just define it as `["Not Provided", "Not Provided"]`

### What areas of the application does this impact?

Collection Details Page: /search/granules/collection-details
Granule Search Page: /search/granules

# Testing

### Reproduction steps

- **Environment for testing:**
- **Collection to test with:**

1. Go to the main page
2. Search for C2450786986-ASF.  This is the "Global Sentinel-1 Burst ID Map".
3. Click on the i to open the collection details.
4. The cards under granules in the left-side panel, should displa "Not Provided" as the Start and End Values.
5. Scroll to the bottom of the granule highlights/cards and click "View Granules" (which will open up the Granule Search page)
6. The granule cards should display "Not Provided" as the Start and End Values.

### Attachments
<img width="1787" alt="Screen Shot 2022-10-18 at 9 41 22 AM" src="https://user-images.githubusercontent.com/4313463/196446517-52982077-5eee-4f2c-bd00-c96ad4fe3c31.png">

<img width="1787" alt="Screen Shot 2022-10-18 at 9 41 43 AM" src="https://user-images.githubusercontent.com/4313463/196446589-47c8d32c-cfab-42a5-b081-b9fc3d76fbc1.png">


# Checklist
